src/main-settings.js

```diff
@@ -23,26 +23,24 @@ import store from './store'
 import Vue from 'vue'
 
 import Nextcloud from './mixins/Nextcloud'
+import { loadState } from '@nextcloud/initial-state'
 
 Vue.mixin(Nextcloud)
 
-const initialStateElement = document.getElementById('twofactor-u2f-initial-state')
-if (initialStateElement) {
-	const devices = JSON.parse(initialStateElement.value)
-	devices.sort()
-	devices.sort((d1, d2) => {
-		if (!d1.name) {
-			return 1
-		} else if (!d2.name) {
-			return -1
-		} else {
-			return d1.name.localeCompare(d2.name)
-		}
-	})
-	store.replaceState({
-		devices
-	})
-}
+const devices = loadState('twofactor_u2f', 'devices');
+devices.sort()
```

Comment:
It was there before but doesn't really make sense?! Could you remove that line please? :)

Reply:
fair enough